api/python/tests/integration/test_packages.py

```diff
@@ -531,8 +534,13 @@ def test_load_into_quilt(self):
         remote_pkg.set('foo2', test_file3)
 
         with patch('time.time', return_value=1234567891), \
+             patch('quilt3.packages.DISABLE_TQDM', True), patch('quilt3.data_transfer.DISABLE_TQDM', True), \
              patch('quilt3.data_transfer.s3_transfer_config.max_request_concurrency', 1):
-            remote_pkg.push('Quilt/package', 's3://my_test_bucket/')
+            stderr = io.StringIO()
+
+            with redirect_stderr(stderr), patch('quilt3.packages.DISABLE_TQDM', True):
+                remote_pkg.push('Quilt/package', 's3://my_test_bucket/')
+            assert not stderr.getvalue()
```

Comment:
Why are you patching `quilt3.packages.DISABLE_TQDM` in two places?

Reply:
That is to test different behaviors when tqdm is disabled or enabled

Reply:
The two places called `.push` which uses tqdm